nvidia_triton/assets/dashboards/nvidia_triton_overview.json

```diff
@@ -1923,5 +1924,8 @@
     ],
     "layout_type": "ordered",
     "notify_list": [],
-    "reflow_type": "fixed"
+    "reflow_type": "fixed",
+    "tags": [
+        "team:aaawesome-nick-team"
```

Comment:
Any idea where this comes from?

Reply:
None, it just shows as a tag on the dashboard in staging so I mistakenly copied it. Removing it.